### ISSUE TITLE
Android: game crashes when speaking with Gizmo 

### ISSUE THREAD

**Issue by NukeItAll** [alexbatalov/fallout1-ce#26]
Hi Alexbatalov,

1st of all... You are awesome. I love playing fallout on my mobile.

The issue:
game crashes when speaking with Gizmo.
Might be same issue as game crashes when speaking with Tandi?

Cheers

**Comment by alexbatalov** [comment]
Post your savegame please.

**Comment by NukeItAll** [comment]
[SLOT01.zip](https://easyupload.io/7v8zrx)

**Comment by alexbatalov** [comment]
Yes, looks like the same problem (#20). It's already been fixed. Use latest [debug](../actions) version, I'll prepare new release with hot fixes later this week.
